pages/womentechmakers.html

```diff
@@ -9,10 +9,28 @@
 
   <img src="{{ site.baseurl }}{{ site.img_dir }}/women-techmakers-poster-702x315.jpg" alt="Women Techmakers. Connect. Create. Celebrate." class="wtm-splash">
 
-  <h2>Tucson Women Techmakers Hackathon</h2>
+  
+  <h2> Women* Techmakers Tucson Hackathon</h2>
   <h2>November 14 - 15, 2015</h2>
 
-  <p>Actually pickled gentrify, ennui fashion axe cardigan normcore letterpress lo-fi PBR chambray narwhal. Tousled American Apparel Portland before they sold out, brunch narwhal cold-pressed lo-fi single-origin coffee Thundercats. Roof party direct trade lo-fi, letterpress sustainable synth taxidermy wayfarers sartorial Portland aesthetic skateboard lomo vinyl retro. Meh pug artisan flexitarian Schlitz beard. Tofu wolf cornhole, hashtag next level narwhal 3 wolf moon Williamsburg dreamcatcher Portland. Readymade Austin meh, migas raw denim VHS Etsy chambray Shoreditch. Intelligentsia fap quinoa biodiesel Neutra, tote bag vegan sriracha church-key lumbersexual aesthetic heirloom.</p>
+  <p>Google Development Group, Code for Tucson, and UA Libraries are proud to present the <b> Women Techmakers Tucson Hackathon </b> on November 14-15 at the University of Arizona's Science and Engineering Library.
+
+<p> Join us for a weekend of food, fun, learning, and hacking! This free event provides an opportunity for women of all levels of technical expertise to collaboratively develop tech-driven projects over the course of two days. All women are encouraged to attend--no computer programming experience required! </p>
+
+<p> <b> Event highlights: </b> </p>
```

Comment:
I'm not sure if we have this as an example elsewhere, which would be a poor example, but most developers prefer not to use `<b />` and `<i />` anymore, favoring the for _semantically_ opinionated `<strong />` and `<em />` respectively.

"strong" is bold by default
"_em_ phasis" is italicized by default